### PR TITLE
add organization

### PR DESCRIPTION
**Pull request by jgsqware**
Signed-off-by: Julien Garcia Gonzalez <julien@giantswarm.io>

related to: https://github.com/giantswarm/giantswarm/issues/2062

I add the organization field in the TPR

### CODE REVIEW

File: spec_test.go [hunk 1]
```diff
@@ -25,6 +25,9 @@ func TestSpecYamlEncoding(t *testing.T) {
 		IPSANs: []string{
 			"172.31.0.1",
 		},
+		Organizations: []string{
+			"system:masters",
+		},
```
Comment (xh3b4sd) [inline spec_test.go:30]: @puja108 just to make sure, this is how it works right?

**Review by xh3b4sd — APPROVED** [review]
LGTM, lets wait what Puja says.

File: spec_test.go (hunk 1 above)
Comment (jgsqware) [reply spec_test.go:30]: Tested for k8s-addons and works. Good for you @xh3b4sd ?

Comment (xh3b4sd) [reply spec_test.go:30]: Good enough for me.

**Pull request merged**